tests/dbus/_misc.py

```diff
@@ -69,5 +73,17 @@ class ServiceR(ServiceABC):
     def setUp(self):
         self._stratisd = subprocess.Popen([os.path.join(_STRATISD), '--sim'])
 
+        # It takes some time for the service to be ready to use, lets wait
```

Comment:
You could add an additional commit that removes the class ServiceABC and changes other things as appropriate. ServiceABC exists because at one point we were running both a C version of stratisd and a Rust version. Those days are long past, so I think that code is entirely obsolete.

Then you could remove this comment, and instead add header comments ```ServiceR```'s ```setUp``` and ```tearDown``` methods that include whatever it is the user needs to know about these methods.

For ```setUp``` something like:

```
Waits up to 10 seconds for stratisd daemon to be ready, checking every 100ms.
```